opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -13,25 +13,41 @@
  */
 package opentracing;
 
+
 /**
- * Tracer is a simple, thin interface for Span creation.
+ * Tracer is a simple, thin interface for Span creation, and Span propagation into different transport formats.
  */
 public interface Tracer {
 
   /**
-   * Create, start, and return a new Span with the given `operationName`, all without specifying a
-   * parent Span that can be used to incorporate the newly-returned Span into an existing trace.
+   * Create, start, and return a new Span with the given `operationName`.
+   * An optional parent Span can be specified used to incorporate the newly-returned Span into an existing trace.
    *
    * <p>Example:
    * <pre>{@code
    * Tracer tracer = ...
-   * Span feed = tracer.startTrace("GetFeed");
-   * Span http = tracer.startTrace("HandleHTTPRequest")
+   * Span feed = tracer.startTrace("GetFeed", null);
+   * Span http = tracer.startTrace("HandleHTTPRequest", feed)
    *                   .setTag("user_agent", req.UserAgent)
    *                   .setTag("lucky_number", 42);
    * }</pre>
    */
-  Span startTrace(String operationName);
+  Span startSpan(String operationName, /* @Nullable */ Span parent);
```

Comment:
^I said I'd add the tags but first…

Once I started to code this it's already pretty obvious how more intuitive (and needed) the builder pattern is. Rather than add it to this PR I've spawned it off as a new https://github.com/opentracing/opentracing-java/pull/12.

The Builder pattern has my vote. But it depends on how the Specification can be interpreted. If there's any objections I'll add the map as I said I would.